docs/quickstarts/flink_processor_cli_quickstart.md

```diff
@@ -5,6 +5,13 @@ Commandline Tool to a standalone Flink cluster. The Feathub job simply consumes
 the data from the Flink datagen connector, computes some feature, and prints out the 
 result.
 
+### Prerequisites
+
+In addition to the prerequisites listed in [README](../../README.md#Prerequisites). The
```

Comment:
The prerequisites referenced here are needed for building python packages. Do we really need all those prerequisites in order to submit FeatHub job via the Flink CLI tool?

For example, is `Maven` needed to use the Flink CLI tool?